walker.go

```diff
@@ -51,15 +49,18 @@ const (
 	countHashes      = "file-hash-count"
 )
 
+// WriteCallback is a function to write a Walk.
+type WriteCallback func(*fspb.Walk) error
+
 // WalkerFromPolicyFile creates a new Walker based on a policy path.
-func WalkerFromPolicyFile(ctx context.Context, path, outpath string, verbose bool) (*Walker, error) {
+func WalkerFromPolicyFile(ctx context.Context, path string, write WriteCallback, verbose bool) (*Walker, error) {
```

Comment:
Since you're modifying the function signature, I wonder if it's better to remove all arguments from here except `ctx` and `path` at this point. It'll possibly be less of a breaking change in the future because users would set the exported fields separately like so:

```go
wk, err := fswalker.WalkerFromPolicyFile(ctx, "path/to/policy")
if err != nil {
  log.Fatal(err)
}
wk.Verbose = true
wk.RunCallback = func(walk *fspb.Walk) error {
  // do something with the walk
}
if err := wk.Run(ctx); err != nil {
  log.Fatal(err)
}
```

Reply:
Sgtm, thanks.